main.cpp

```diff
@@ -1,53 +1,114 @@
-#include <SFML/Graphics.hpp>
```

Comment:
Aceeasi problema cu line ending-urile
Mi-ai redenumit functia, si nu e chiar corect, pentru ca aia face doar drawing-ul. Schimb-o la loc in loadBoard.
Schimbarile de aici au conflicte cu ce e pe master acum. Din fericire ai facut functii, si poti schimba repede. Dar fa-o.
De asemenea, fisierele din bin, obj, si orice dll n-au ce cauta pe git. pentru bin si obj am specificat ceva in .gitignore, cred ca le-ai bagat tu manual. Scoate-le